packages/core/src/agent/didcomm/versions/v2/index.ts

```diff
@@ -0,0 +1,25 @@
+import type { DecryptedMessageContext, EncryptedMessage, SignedMessage, MessageType } from '../../types'
+import type { AgentContext } from './../../../context/AgentContext'
+import type { DIDCommV2Message } from './DIDCommV2Message'
+
+export { DIDCommV2Message } from './DIDCommV2Message'
+export { DIDCommV2BaseMessage, DIDComV2BaseMessageConstructor } from './DIDCommV2BaseMessage'
+
+export interface PackMessageParams {
+  toDID?: string
+  fromDID?: string
+  signByDID?: string
+  serviceId?: string
+  forward?: boolean
+  type?: MessageType
```

Comment:
Can we make these a bit more explicit. What does forward mean in this case? Do we want to forward the message? To whom? Or is it whether it should skip adding forward envelope messages for each hop? 

Also type, is this the message type (like https://didcomm.org/connections/1.0/invitation) which the name suggests to me, however it seems like it is more the type of packing / encryption /envelope that should be used. Can we make the property name (type) and the type name (MessageType) more descriptive?